### PR TITLE
Unify timeout constants ROBOT_CRASH_TIMEOUT_S and ROBOT_FATAL_TIMEOUT_S (fixes #3023)

### LINKED ISSUE #3023: [Thunderscope] Unify ROBOT_CRASH_TIMEOUT_S and ROBOT_FATAL_TIMEOUT_S
### Description of the task

We have two similar constants called ROBOT_FATAL_TIMEOUT_S in `software/thunderscope/log/g3log_widget.py` and `ROBOT_CRASH_TIMEOUT_S` in `software/thunderscope/robot_diagnostics/robot_view.py`. It makes sense to unify those two constants into one constant.

### Acceptance criteria

- [ ] Make a new constant in `software/thunderscope/constants.py`
- [ ] Use these constants in g3log_widget.py and robot_view.py
- [ ] Remove the old ROBOT_CRASH_TIMEOUT_S and ROBOT_FATAL_TIMEOUT_S

### PR DESCRIPTION
**Pull request by aviwad**
### Description

`ROBOT_FATAL_TIMEOUT_S` in `software/thunderscope/log/g3log_widget.py` and `ROBOT_CRASH_TIMEOUT_S` in `software/thunderscope/robot_diagnostics/robot_view.py` have been replaced with a new constant `ROBOT_CRASH_TIMEOUT_S` set in `software/thunderscope/constants.py`

### Testing Done

I built and ran thunderscope by cherrypicking my recent commit onto Aden's arm64 branch. However exhaustive testing was not possible because I can only compile @AdenC123's arm64 branch on my M1 mac. 

### Resolved Issues

Resolves #3023 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ x] **Remove all commented out code**
- [ x] **Remove extra print statements**: for example, those just used for testing
- [ x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/thunderscope/constants.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ class IndividualRobotMode(IntEnum):
 # in robot communications
 ROBOT_COMMUNICATIONS_TIMEOUT_S = 0.02
 
+# ignore repeated crash proto
```
Comment (williamckha) [inline src/software/thunderscope/constants.py]: Can we make this comment more descriptive? e.g. 
```suggestion
# Max time (in seconds) tolerated between repeated crash protos until
# crash alert occurs
```

**Review by williamckha — COMMENTED** [review]
LGTM, just one suggestion

**Comment by aviwad** [comment]
> LGTM, just one suggestion

Committed!

**Review by williamckha — DISMISSED** [review]

**Review by nimazareian — DISMISSED** [review]
LGTM!

**Comment by nimazareian** [comment]
Just pull upstream master into your branch, so CI passes and you can merge

**Comment by aviwad** [comment]
I rebased this branch to upstream/master and for some reason the commits duplicated. Going to try undoing this to clean up the commits.

Edit: I think it's fixed now.

**Review by williamckha — APPROVED** [review]

**Comment by williamckha** [comment]
Merge is probably easier than rebase for us. We don't really reap the benefits of rebasing since all commits are squashed before merging into master, so having a clean commit history doesn't matter

**Review by nimazareian — APPROVED** [review]

**Comment by itsarune** [comment]
@aviwad this PR is ready to merge

**Pull request merged**